src/main/java/com/lunark/lunark/controller/RecordController.java

```diff
@@ -0,0 +1,22 @@
+package com.lunark.lunark.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/records")
+public class RecordController {
+
+    @GetMapping("/download")
+    public ResponseEntity<String> downloadRecord(@RequestParam Long recordId) {
```

Comment:
Shouldn't this method take the same parameters as the one for getting a record?

Reply:
I'm not sure how this will work.

Reply:
The method signature would be public ResponseEntity<String> downloadRecord    public ResponseEntity<String> downloadRecord(@RequestParam Long recordId)